app/views/layouts/application.html.erb

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+
+    <title><%= content_for?(:title) ? yield(:title) : "Edamammmm" %></title>
+
+    <%= stylesheet_link_tag    "application" %>
+    <%= javascript_include_tag "application", 'data-turbolinks-track' => true %>
+    <%= csrf_meta_tags %>
+    <script src="https://developer.edamam.com/attribution/badge.js"></script>
+  </head>
+  <header>
+    <%= link_to(image_tag("logo.jpg", alt: "Site logo"), "/" , class: "logo") %>
+    <nav>
+      <%= form_tag recipes_path(@label), method: "get" do %>
+        <%= label_tag(:message, "Enter Recipe name") %>
+        <%= text_field_tag(:message) %>
+        <%= submit_tag("Search") %>
+
+      <% end %>
+
+    </nav>
+  </header>
+  <body>
```

Comment:
The `<header>` and `<nav>`, and all other content that appears on the page, should go inside the `<body>` tag.